components/MapComponent.tsx

```diff
@@ -1,19 +1,27 @@
+import { ImageSource } from "aws-sdk/clients/lookoutvision"
 import { Location } from "lib/location/Location"
 import React, { useState } from "react"
-import { StyleProp, View, ViewStyle } from "react-native"
-import MapView, { Marker, PROVIDER_GOOGLE } from "react-native-maps"
+import { StyleProp, StyleSheet, View, ViewStyle } from "react-native"
+import MapView, { Circle, Marker, PROVIDER_GOOGLE } from "react-native-maps"
 
 // Type for the map markers, in order to separate their logic out
-export interface mapMarker {
+export interface MapMarker {
   key: string
   title: string
```

Comment:
Color and icon will be set by the user while creating an event, so I'd think they would be included in the event data type @mhayes853 Also I don't think we have color in the event schema yet so ask Nico or Tristan about adding that to the backend.